pages/dashboard/views/index.jsx

```diff
@@ -21,6 +21,7 @@ const Index = ({
             establishments.map(est =>
               <li key={est}>
                 <Link page="establishment.dashboard" establishment={ est.id } label={ est.name } />
+                <br />
```

Comment:
Changing the styling on `ul.dashboard a` removed the `display: block` so this is just a temporary fix until we decide what the home dashboard looks like.

Reply:
Can we wrap in `<p>` instead of inserting `<br/>` please? It's _mildly_ more semantic.

Reply:
done.